src/app/components/loading-gif/loading-gif.component.ts

```diff
@@ -0,0 +1,11 @@
+import {Component, Input} from '@angular/core';
+
+@Component({
+  selector: 'loading-gif',
+  templateUrl: './loading-gif.component.html',
+  styleUrls: ['./loading-gif.component.scss']
+})
+export class LoadingGifComponent {
```

Comment:
No matter if we are using `.gif`-file or an animated icon font for the loading indicator I think we should perhaps rename `LoadingGifComponent` to `LoadingCompontent` then the name wont be "bound" to the concrete implementation (we could then refactor gif => icon without any "need" to update the component name).